src/functions.php

```diff
@@ -1,33 +1,79 @@
 <?php
 
-function getAllGames(object $db): array
+function getAllGames(PDO $db): array
 {
     $query = $db->prepare(
-        'SELECT `name`, `genre`.`genre`, `platform`.`platform`, `image`, `agerating`.`age` FROM videogames
+        'SELECT `name`, `genre`.`genre`, `platform`.`platform`, `image`, `agerating`.`agerating` FROM videogames
         JOIN `genre` ON `videogames`.`genreid` = `genre`.`id`
         JOIN `platform` ON `videogames`.`platformid` = `platform`.`id`
         JOIN `agerating` ON `videogames`.`ageid` = `agerating`.`id`;'
     );
     $result = $query->execute();
-    if ($result) {
-        $videogames = $query->fetchAll();
-    } else {
+    if (!$result) {
         throw new ErrorException("Couldn't retrieve information from database");
+    } else {
+        $videogames = $query->fetchAll();
     }
     return $videogames;
 }
 
+function checkImgExists(array $game): string {
+    if (!$game['image']) {
+        return "";
+    } else {
+        return "<img class='card-img' src='{$game['image']}' alt='Image of {$game['name']} cover'>";
+    }
+}
 function displayAllGames(array $games): string
 {
     $result = "";
     foreach ($games as $game) {
+        $img = checkImgExists($game);
         $result .= "<div class='card'>
```

Comment:
Instead of <p><strong> you could style your card class .card p{} which could be used to turn all the p's strong!